x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/expression/TypeResolutions.java

```diff
@@ -117,58 +117,6 @@ public static TypeResolution isIPAndExact(Expression e, String operationName, Pa
         return isExact(e, operationName, paramOrd);
     }
 
-    public static TypeResolution isFoldable(Expression e, String operationName, ParamOrdinal paramOrd) {
```

Comment:
As this shouldn't be used on type resolution, I don't think we should have it here. QL has it here, but now with esql-core we can be more strict